File: Python/1036.py
# -*- coding: utf-8 -*-
import math

a, b, c = input().split(" ")

a = float(a)
b = float(b)
c = float(c)
delta = pow(b, 2) - (4 * a * c)

if a == 0 or delta <= 0:
    print("Impossivel calcular")

else:
    r1 = (- b + math.sqrt(delta)) / (2 * a)
    r2 = (- b - math.sqrt(delta)) / (2 * a)
    print("R1 = %.5f" % r1)
    print("R2 = %.5f" % r2)

File: Java/1036.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        
        double a = leitor.nextDouble();
        double b = leitor.nextDouble();
        double c = leitor.nextDouble();
        double delta = Math.pow(b, 2) - (4 * a * c);
        double r1;
        double r2;
        
        if (a == 0 || delta <= 0) {
            System.out.println("Impossivel calcular");
        }
        else {
            r1 = (- b + Math.sqrt(delta)) / (2 * a);
            r2 = (- b - Math.sqrt(delta)) / (2 * a);
            System.out.printf("R1 = %.5f\n" +
                              "R2 = %.5f\n" , r1, r2);
        }
    }
 
}